client/landing/stepper/declarative-flow/internals/steps-repository/bundle-confirm/index.tsx

```diff
@@ -285,7 +287,7 @@ const BundleConfirm: Step = function BundleConfirm( { navigation } ) {
 
 	const headerText = __( 'One final step' );
 	const subHeaderText = __(
-		'We’ve highlighted a few important details you should review before we create your store.'
+		"We've highlighted a few important details you should review before we create your store."
```

Comment:
Nit: we should push this into its own PR as it may trigger downstream translations, which is definitely not the point of this PR.